pymoab/core.pyx

```diff
@@ -93,6 +93,13 @@ cdef class Core(object):
         cdef moab.ErrorCode err
         cdef Range r
         cdef np.ndarray[np.uint64_t, ndim=1] arr
+        cdef moab.DataType type
```

Comment:
`type` is a Python builtin so it is typical to avoid using it as a variable name. `t`, `typ`, and `mtype` would all work here.